nevergrad/functions/images/imageLosses.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from koncept.models import Koncept512
+
+
+class _ImageLoss:
+    def __init__(self) -> None:
+        pass
+
+    def __call__(self, img: np.ndarray) -> float:
+        assert False
```

Comment:
A reason you don't need this is that this is basically `tp.Callable[[np.ndarray], float]`